content/docs/guides/node-exporter.md

```diff
@@ -62,13 +62,14 @@ curl http://localhost:9100/metrics | grep "node_"
 
 ## Configuring your Prometheus instances
 
-Your locally running Prometheus instance needs to be properly configured in order to access Node Exporter metrics. The following [`scrape_config`](../../prometheus/latest/configuration/configuration/#scrape_config) block (in a `prometheus.yml` configuration file) will tell the Prometheus instance to scrape from the Node Exporter via `localhost:9100`:
+Your locally running Prometheus instance needs to be properly configured in order to access Node Exporter metrics. The following [`scrape_config`](../../prometheus/latest/configuration/configuration/#scrape_config) block (in a `prometheus.yml` configuration file) will tell the Prometheus instance to scrape, and how frequently, from the Node Exporter via `localhost:9100`:
 
 <a id="config"></a>
 
 ```yaml
 scrape_configs:
-- job_name: 'node'
+- job_name: node
+  scrape_interval: 15s
```

Comment:
We should be pointing users at the global setting rather than doing per-job config overrides.

Reply:
Sure, I also prefer complete example configs.